medusa/search/backlog.py

```diff
@@ -49,6 +49,7 @@ def __init__(self):
         self.amActive = False
         self.amPaused = False
         self.amWaiting = False
+        self.force = False
```

Comment:
Why do you make a new attribute for this? Just pass it to self.search_backlog(force)